File: frameworks/test.web/src/main/java/com/example/scaffold/frameworks/test/web/RequestParameters.java
package com.example.scaffold.frameworks.test.web;

import java.util.Map;

public class RequestParameters {

    private final Map<String, ?> variables;

    public static RequestParameters parameters(Map<String, ?> variables) {
        return new RequestParameters(variables);
    }

    public RequestParameters(Map<String, ?> variables) {
        this.variables = variables;
    }

    public Map<String, ?> asMap() {
        return variables;
    }

}

File: frameworks/test.web/src/main/kotlin/com/qomolangma/frameworks/test/web/RequestParameters.kt
package com.qomolangma.frameworks.test.web

class RequestParameters(private val variables: Map<String, *>) {
    fun asMap(): Map<String, *> {
        return variables
    }

    companion object {
        @JvmStatic
        fun parameters(variables: Map<String, *>): RequestParameters {
            return RequestParameters(variables)
        }
    }
}
